action.yml

```diff
@@ -4,6 +4,39 @@
   branding:
     icon: 'aperture'
     color: 'blue'
+  inputs:
+    installDir:
+      description: 'Installation folder of Parasoft SOAtest. If not specified, the soatestcli executable must be added to PATH.'
+      required: false
+    workingDir:
+      description: 'Working directory for running SOAtest.'
+      required: false
+      default: ${{ github.workspace }}
+    testConfig:
+      description: 'Test configuration to be used for test execution.'
+      required: false
+      default: 'user://Example Configuration'
+    resource:
+      description: 'Path to the test suite(s) relative to the workspace.'
```

Comment:
I see "relative to the workspace" in many of the input variables here. Are we sure it could only be "relative" path?
If not, please change to: "The path to the test suite(s) to run."
And I'll left comments for improving other descriptions without relative path for now.